_posts/2017-08-26-a11y-and-other-numeronyms.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: |
+    Quick Tip: A11y and a brief numeronyms primer
+description: The numeronym "a11y" is shorthand for "accessibility"
+author: Scott O'Hara
+date: 2017-08-26
+# last_updated:
+published: true
+categories:
+  - Quick Tips
+---
+You may have seen "a11y" used as a [hash tag on Twitter](https://twitter.com/search?q=%23a11y), or used in a comment, article, and since you're reading this, likely in the name of a certain website about accessibility, ahem... But if you're unfamiliar with the term "[numeronym](https://en.wikipedia.org/wiki/Numeronym)", you may not have realized that a11y isn't just a hash tag, nor is it a hip way to spell <em>ally</em>. Rather a11y is a shorthand for <em>accessibility</em>.  
```

Comment:
>, or used in a comment, article, and since you're reading this, likely in the name of a certain website about accessibility, ahem...
👇
> , used in a comment thread, or written in an article. Since you're reading this right now, you've likely seen it in the name of a certain website about accessibility. <em>*ahem…*</em>